README.md

```diff
@@ -16,8 +16,12 @@ Although the `flakehub-push` Action requires little configuration, you may benef
 
 ## Integration
 
-This action sets the `flakeref` output to the exact name and version that was published.
-The flake reference can be used in subsequent steps or workflows as part of a deployment process.
+This action sets outputs for integrating into CD pipelines:
```

Comment:
Let's make this a table with rows for name, description, and example

Reply:
Done!